types/async-retry/index.d.ts

```diff
@@ -1,17 +1,27 @@
-// Type definitions for async-retry 1.1
+// Type definitions for async-retry 1.2
 // Project: https://github.com/zeit/async-retry#readme
 // Definitions by: Albert Wu <https://github.com/albertywu>
+//                 Pablo Rodríguez <https://github.com/MeLlamoPablo>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-export function retry<A>(fn: RetryFunction<A>, opts: Options): Promise<A>;
+declare function AsyncRetry<A>(
+	fn: AsyncRetry.RetryFunction<A>,
+	opts: AsyncRetry.Options
+): Promise<A>;
 
-export interface Options {
-    retries?: number;
-    factor?: number;
-    minTimeout?: number;
-    maxTimeout?: number;
-    randomize?: boolean;
-    onRetry?: (e: Error) => any;
+declare namespace AsyncRetry {
+	function retry<A>(fn: RetryFunction<A>, opts: Options): Promise<A>;
```

Comment:
Looking [at the source](https://github.com/zeit/async-retry/blob/master/lib/index.js), I don't see that `retry` is exported with a separate name. So I think it might be correct to delete this &mdash; I note that the test no longer uses this either.

Reply:
You're right, thanks for pointing that out.